extensions/amp-lightbox/amp-lightbox.md

```diff
@@ -102,6 +109,47 @@ The `amp-lightbox` exposes the following actions you can use [AMP on-syntax to t
   </tr>
 </table>
 
+## <a id="a4a"></a> Using `amp-lightbox` in AMPHTML ads
+
+{% call callout('Note', type='note') %}
+The `amp-lightbox` component for use in AMPHTML ads is [experimental](https://www.ampproject.org/docs/reference/experimental) and under active development. To use `amp-lightbox` in AMPHTML ads, [enable the `amp-lightbox-a4a-proto` experiment](http://cdn.ampproject.org/experiments.html).
+{% endcall %}
+
+There are some differences between using `amp-lightbox` in normal AMP documents vs. [ads written in AMPHTML](../amp-a4a/amp-a4a-format.md):
+
+### Requires close-button
+
+For AMPHTML ads, the `close-button` attribute is required. This attribute causes a header to render at the top of your lightbox. The header contains a close button and a label that displays "Ad". Requirement of this header is needed to:
+
+- Set a consistent and predictable user experience for AMPHTML ads.
+- Ensure that an exit point for the lightbox always exists, otherwise the creative could effectlively hijack the host document content via a lightbox.
+
+The `close-button` attribute is required and only allowed in AMPHTML ads. In regular AMP documents, you can render a close button wherever you need it as part of the `<amp-lightbox>` content.
+
+### Scrollable lightboxes are disallowed
+
+For AMPHTML ads, scrollable lightboxes are not allowed.
+
+The background of the `<body>` element for AMPHTML ads that contain a lightbox is also set to transparent. For more information, see the [Transparent background](#transparent-background) section below.
+
+#### Transparent background
+
+When you use `<amp-lightbox>` in AMPHTML ads, the background of your `<body>` element becomes transparent because the AMP runtime resizes and realigns your creative's content before the lightbox is expanded. This is done to prevent a visual "jump" of the creative while the lightbox opens. If your creative needs a background, set it on an intermediate container (like a full-size `<div>`) instead of the `<body>`.
+
+When the AMPHTML ad is running in a third-party environment (for example, AMP inabox), the creative is centered relative to the viewport and is then expanded. This is because third-party iframes need to rely on a postMessage API to enable features like frame resizing, which is asynchronous, so centering the creative first allows a smooth transition without visual jumps.
```

Comment:
What is "AMP inabox"?  There are no docs, and should be if you are referencing it.  Please add link to relevant information.